assets/src/edit-story/components/library/libraryTabs.js

```diff
@@ -14,30 +14,55 @@
  * limitations under the License.
  */
 
+/**
+ * WordPress dependencies
+ */
+import { useRef, useCallback } from '@wordpress/element';
+
 /**
  * Internal dependencies
  */
+import { useKeyDownEffect } from '../keyboard';
 import useLibrary from './useLibrary';
-import { Tabs, Media, Text, Shapes, Links } from './tabs';
+import { Tabs, getPanes } from './panes';
+
+function getTabId(tab) {
+  return `#library-tab-${tab}`;
+}
 
 function LibraryTabs() {
   const {
     state: { tab },
     actions: { setTab },
-    data: {
-      tabs: { MEDIA, TEXT, SHAPES, LINKS },
-    },
+    data: { tabs },
   } = useLibrary();
-  const tabs = [
-    [MEDIA, Media],
-    [TEXT, Text],
-    [SHAPES, Shapes],
-    [LINKS, Links],
-  ];
+  const panes = getPanes(tabs);
+  const ref = useRef();
+  const handleNavigation = useCallback(
+    (isPrevious) => () => {
```

Comment:
Nit: but I usually find it easier when it's a `dir` number with 1 and -1 values.

Reply:
Fixed in f7756942.